Qadriyah/output8-1
# #164460804 Setup React Router

**What does this PR do?**

Sets up React router and the Integrations

**Description of Task to be completed?**

- Install react-router-dom
- Apply a Router to the App
- Setup TravisCI integration
- Setup codeclimate
- Setup Hound 
- Setup Eslint
- Add budges to repo

**How should this be manually tested?**

- Run `npm install` to install dependencies.
- Run` npm start` to run the application.
- On github, navigate to the develop branch to see the budges

**Any background context you want to provide?**

**What are the relevant pivotal tracker stories?**

[#164460804](https://www.pivotaltracker.com/n/projects/2317430/stories/164460804)
[#164460721](https://www.pivotaltracker.com/n/projects/2317430/stories/164460721)
